backend/build.gradle

```diff
@@ -20,9 +20,9 @@ dependencies {
     implementation 'com.amazonaws:aws-java-sdk-dynamodb'
 
     api project(path: ':model')
+    implementation 'com.google.firebase:firebase-database:20.0.4'
```

Comment:
I'm not sure I see the need for a second firebase database.

Reply:
Agreed and will be changed.